server/project/accounts/models.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# Create your models here.
+
+from django.db import models
+from django.utils import timezone
+
+ACCOUNT_TYPES = (('a', 'Asset'), ('l', 'Liability'), ('e', 'Equity'))
+BOOL_TYPES = (('t', 'True'), ('f', 'False'), ('n', 'Null'))
+
+
+class Account(models.Model):
+    account_id = models.IntegerField(primary_key=True)
+    account_name = models.CharField(max_length=60, unique=True)
+    priority = models.IntegerField(verbose_name="Priority (0 is top)", default=0)
+    is_active = models.CharField(max_length=1, choices=BOOL_TYPES, default='t')
+    category = models.CharField(max_length=1, choices=ACCOUNT_TYPES, default='a')
+    starting_value = models.DecimalField(max_digits=20, decimal_places=2, default=0)
+    comments = models.CharField(max_length=200, null=True)
+    date_created = models.DateTimeField(default=timezone.now())
+
+    def __str__(self):
+        return "{:03}: ".format(self.account_id) + self.account_name + " ${:.2f}".format(self.get_balance())
+
+    def get_side(self):
+        return "Left" if (self.category == 'a') else "Right"
```

Comment:
Can you move Left and Right into an enumeration? I imagine we'll want to reference the side in our business logic in the future so it would be best to have it in an enumeration that can be imported where needed.

Reply:
Removed as unneeded. is_asset() provides all the information needed.